src/sql/src/plan/query.rs

```diff
@@ -4330,10 +4330,10 @@ fn plan_function<'a>(
             )
         }
         Func::Aggregate(_) => {
-            sql_bail!("aggregate functions are not allowed in {}", ecx.name);
+            sql_bail!("aggregate functions are not allowed in {} (function {})", ecx.name, unresolved_name);
```

Comment:
Could we update the sqllogictests to test for this new pattern? It looks like we [already have a few](https://github.com/search?q=aggregate+functions+are+not+allowed+repo%3AMaterializeInc%2Fmaterialize+extension%3Aslt&type=Code&ref=advsearch&l=&l=) that could be updated.